using-kontena/stack-file.md

```diff
@@ -147,11 +147,21 @@ There are many configuration options available for defining Kontena Services. In
   * **`interval`** - The interval of automatic redeployment of the service. This can be used as an "erosion-resistance" mechanism. Format <number><unit>, where unit = min, h, d. For example, value `7d`.
 * **`affinity`** - Specify affinity rules that will be used by Kontena Platform scheduler when scheduling this Kontena Service. Affinity rules may be positive (`==`) or negative (`!=`) and they may be compared against Kontena Node name, Kontena Service name, containers or labels. See usage [example](#using-affinity-rules).
 * **`hooks`** - Specify a list of commands that are executed at various stages of this Kontena Service lifecycle. The currently supported stages are `post_start` and `pre_build`. See usage [example](#using-hooks).
```

Comment:
We have more stages now so "The currently supported stages are `post_start` and `pre_build`." needs updating.

Reply:
Fixed.